Runtime/Cesium3DTileset.cs

```diff
@@ -39,6 +39,7 @@ public enum CesiumDataSource
     /// </remarks>
```

Comment:
Since this is a user-facing change, I think it's worth mentioning in `CHANGES.md`. Could you update it to mention this change, following the format of the other change log entries?

Under the "Additions" section, this can say something like: Cesium components now appear under the "Cesium" category in the Component menu.

Reply:
Done.